cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -714,13 +612,13 @@ tag(['upstream'], () => {
           });
       });
     }
-
-    it('can create a new credential, assign a team and apply role(s)', () => {
+    it('create a new credential, assign a team and apply role(s)', () => {
```

Comment:
this change is not a proper sentence. I feel it should stay as it was written. 'It can create a new credential..." as opposed to 'It create a new credential...'

Reply:
I just revert tests, and goal was to minimize changes to keep traceability. We can add a follow up PR.


Goal of this PR is not to fix current errors, but address old code that was merged.